front/tests/lib/datasets.test.ts

```diff
@@ -1,4 +1,5 @@
 import { checkDatasetData, getDatasetTypes } from "@app/lib/datasets";
+import { describe, test, expect } from "@jest/globals";
```

Comment:
Adding the following to .eslintrc.json should be enough when "no-undef" is enabled

"env": {
    "jest": true
}

Reply:
Thank you !